snapcraft.yaml

```diff
@@ -278,7 +287,10 @@ parts:
       [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && [ "$(uname -m)" != "ppc64le" ] && [ "$(uname -m)" != "s390x" ] && [ "$(uname -m)" != "armv7l" ] && exit 0
       set -ex
 
-      make USERCFLAGS=-Wno-error=format-truncation criu
+      git config user.email "noreply@lists.canonical.com"
+      git config user.name "LXD snap builder"
```

Comment:
Unneeded as we don't do any cherry-picking or such, but also harmless.

Reply:
thanks, yeah was trying to sync as much as possible.